### PR TITLE
Enhanced @bindable

### PR DESCRIPTION
**Pull request by lbwexler**
Hoist P/R Checklist
-------------------

**Pull request authors:** Review and check off the below. Items that do not apply can also be
checked off to indicate they have been considered. If unclear if a step is relevant, please leave
unchecked and note in comments.

- [x] Caught up with `develop` branch as of last change.
- [x] Added CHANGELOG entry, or determined not required.
- [x] Reviewed for breaking changes, added `breaking-change` label + CHANGELOG if so.
- [x] Updated doc comments / prop-types, or determined not required.
- [x] Reviewed and tested on Mobile, or determined not required.
- [x] Created Toolbox branch / PR, or determined not required.

**If your change is still a WIP**, please use the "Create draft pull request" option in the split
button below to indicate it is not ready yet for a final review.

> **Pull request reviewers:** when merging this P/R, please consider using a *squash commit* to
> collapse multiple intermediate commits into a single commit representing the overall feature
> change. This helps keep the commit log clean and easy to scan across releases. PRs containing a
> single commit should be *rebased* when possible.

### CODE REVIEW

File: mobx/decorators.js [hunk 1]
```diff
@@ -39,10 +40,43 @@ export function settable(target, property, descriptor) {
...
+function createBindable(target, name, descriptor, isRef) {
+
+    // 1) set up a set function, if one does not exist.  This was the original side-effect of
+    // bindable and still used for backward compatibility by HoistBase.setBindable.
+    const setterName = 'set' + upperFirst(name);
+    if (!target.hasOwnProperty(setterName)) {
+        const value = function(v) {
+            this[name] = v;
+        };
+        Object.defineProperty(target, setterName, {value});
```
Comment (TomTirapani) [inline mobx/decorators.js:63]: How does this ensure the `setXXX` function is an @action and thus works with mobx strict mode?

Comment (lbwexler) [reply mobx/decorators.js:63]: the set function it delegates to on line 61 is actually an action.

Comment (TomTirapani) [reply mobx/decorators.js:63]: Ah yes, of course - this is using the setter defined below 👍

**Comment by lbwexler** [comment]
Reviewed with Tom, ATM, and Jakub

**Pull request merged**